src/command_modules/azure-cli-extension/azure/cli/command_modules/extension/__init__.py

```diff
@@ -61,5 +61,17 @@ def load_arguments(self, command):
             c.argument('source', options_list=['--source', '-s'], help='Filepath or URL to an extension', completer=FilesCompleter())
             c.argument('yes', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation.')
 
+            # Help, because it is suppressed.
+            # Extra URLs of package indexes to use. This should point to a repository compliant
+            # with PEP 503 (the simple repository API) or a local directory laid out in the same format.
+            # Similar to the equivalent flag in pip install. But, the python indexes
+            # specified using --extra-index-url have higher precedence than https://pypi.python.org/simple
+            c.argument('extra_index_urls', options_list=['--extra-index-url'], nargs='+',
+                       help=argparse.SUPPRESS)
```

Comment:
You can have this under the `self.argument_context('extension')` group above then no need to duplicate it below.

Reply:
Resolved.